cdap-gateway/src/main/java/co/cask/cdap/gateway/router/handlers/SecurityAuthenticationHttpHandler.java

```diff
@@ -80,6 +83,23 @@ public SecurityAuthenticationHttpHandler(String realm, TokenValidator tokenValid
     this.discoveryServiceClient = discoveryServiceClient;
     this.discoverables = discoveryServiceClient.discover(Constants.Service.EXTERNAL_AUTHENTICATION);
     this.configuration = configuration;
+    this.bypassPattern = createMatcher(configuration.get(Constants.Security.Router.BYPASS_AUTHENTICATION_REGEX));
```

Comment:
See the CDAP docs: http://docs.cask.co/cdap/3.2.2/en/admin-manual/security.html, particularly: 

CDAP Security is configured in cdap-site.xml and cdap-security.xml:
- cdap-site.xml has non-sensitive information, such as the type of authentication mechanism and their configuration.
- cdap-security.xml is used to store sensitive information such as keystore passwords and SSL certificate keys. It should be owned and readable only by the CDAP user.

In that light it seems to belong into cConf. (Almost everything under Security.Constants is in cConf, not sConf).